pkgs/by-name/fl/flashprog/package.nix

```
@@ -0,0 +1,53 @@
+{ fetchurl
+, stdenv
+, installShellFiles
+, lib
+, libftdi1
+, libgpiod
+, libjaylink
+, libusb1
+, pciutils
+, pkg-config
+, jlinkSupport ? false
+, gpioSupport ? false
+}:
+
+stdenv.mkDerivation rec {
+  pname = "flashprog";
+  version = "1.0.1";
+
+  src = fetchurl {
+    url = "https://flashprog.org/releases/flashprog-v${version}.tar.bz2";
+    hash = "sha256-iIhj38JUqd18AN/ayKMadIecMdtcFjdfs/IWNHwDzHE=";
+  };
+
+  nativeBuildInputs = [ pkg-config installShellFiles ];
+  buildInputs = [ libftdi1 libusb1 ]
+    ++ lib.optionals (!stdenv.isDarwin) [ pciutils ]
+    ++ lib.optional jlinkSupport libjaylink
+    ++ lib.optional gpioSupport libgpiod;
```

```suggestion
    ++ lib.optionals jlinkSupport [ libjaylink ]
    ++ lib.optionals gpioSupport [ libgpiod ];
```

Let's be consistent.